pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -250,13 +252,26 @@ func (s *Service) Delete(ctx context.Context, provider string) error {
 		return err
 	}
 
+	// When deleting settings for SAML, clear the Settings table
+	if s.configurableProviders[social.SAMLProviderName] && provider == social.SAMLProviderName {
```

Comment:
The first check is not necessary here, because in line #241 it has been checked.
```suggestion
	if provider == social.SAMLProviderName {
```

Reply:
Changed as requested